content/04.use-cases-b.md

```diff
@@ -1,17 +1,17 @@
 <!--## GitHub in EcoEvo examples (Part 2)-->
 
-### Hosting a website
+### Hosting a website{#hosting-a-website}
 
 <!--*Contributors to this section: Rob Crystal-Ornelas, Emma Hudgins*   -->
 Personal or laboratory websites can improve the sharing of research findings, build online presence, and increase coordination of research efforts [@doi:10.1038/nj7142-347a].
-Despite researchers in ecology and evolution generally lacking experience in building or hosting webpages, many tools have been developed to help this process.
-Static websites can now be easily built using independent software and languages, such as Jekyll (<https://jekyllrb.com>), Hugo (<https://gohugo.io>), Quarto (<https://quarto.org>), and wowchemy (<https://wowchemy.com>), or with the help of dependencies in the programming languages scientists commonly use, such as the `distill` (<https://github.com/rstudio/distill>) and `blogdown` (<https://bookdown.org/yihui/blogdown>) R packages.
-The resulting files from static websites can then be hosted in repositories, from which one can activate GitHub's Pages (<https://pages.github.com>) feature, allowing for the direct live hosting of HyperText Markup Language (HTML), Cascading Style Sheets (CSS) and JavaScript files within a free github.io domain.
+Despite researchers in ecology and evolution generally lacking experience in building or hosting webpages, many tools have been developed to help this process. 
+Simple websites can consist of a single HTML or markdown document that will be rendered as a website by activating GitHub's Pages (<https://pages.github.com>) feature. 
+More complex static websites can now be easily built from a series of markdown documents using a variety of software including literate programming tools like Quarto (<https://quarto.org>) and R packages such as `distill` (<https://github.com/rstudio/distill>) and `blogdown` (<https://bookdown.org/yihui/blogdown>).
```

Comment:
Maybe remove sentence since:
1) want to cut down on length
2) blogdown _is_ more technical and prob out of scope for this paper